src/components/views/AgreementForm.js

```diff
@@ -127,6 +129,31 @@ class AgreementForm extends React.Component {
     this.setState({ sections });
   }
 
+  toggleAllSections = (expand) => {
+    this.setState((curState) => {
+      const newSections = expandAllFunction(curState.sections, expand);
+
+      return {
+        sections: newSections
+      };
+    });
+  }
+
+  expandAllSections = (e) => {
+    e.preventDefault();
+    this.toggleAllSections(true);
+  }
+
+  collapseAllSections = (e) => {
+    e.preventDefault();
+    this.toggleAllSections(false);
+  }
+
```

Comment:
All this code needs to be cleaned up to use `AccordionStatus` and the `ref` as you had done in the `Agreement` component

Reply:
This is not part of the JIRA issue. The keyboard shortcuts are implemented (as in the example from ui-users).
If we want to change the structure of the AccordionSet should we make this a separate issue?

Reply:
Your call. Honestly, Its straightforwad to move it to the `AccordionStatus` behavior. If you wish to have a separate JIRA, then please feel free to do so and block ERM-1151 on the newly created JIRA.